code/game/machinery/dna_infuser/organ_sets/gondola_organs.dm

```diff
@@ -0,0 +1,138 @@
+
+/obj/item/organ/internal/heart/gondola/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/organ_set_bonus, /datum/status_effect/organ_set_bonus/gondola)
+	AddElement(/datum/element/noticable_organ, "radiates an aura of serenity.")
+
+/obj/item/organ/internal/heart/gondola/Insert(mob/living/carbon/receiver, special, drop_if_replaced)
+	. = ..()
+	if(!(FACTION_HOSTILE in receiver.faction))
+		factions_to_remove += FACTION_HOSTILE
+	if(!(FACTION_MINING in receiver.faction))
+		factions_to_remove += FACTION_MINING
+	receiver.faction |= list(FACTION_HOSTILE, FACTION_MINING)
+
+/obj/item/organ/internal/heart/gondola/Remove(mob/living/carbon/heartless, special)
+	. = ..()
+	for(var/faction in factions_to_remove)
+		heartless.faction -= faction
+	//reset this for a different target
+	factions_to_remove = list()
+
+/// Zen (tounge): You can no longer speak, but get a powerful positive moodlet
+/obj/item/organ/internal/tongue/gondola
+	name = "mutated gondola-tongue"
+	desc = "Gondola DNA infused into what was once a normal tongue."
+	icon = 'icons/obj/medical/organs/infuser_organs.dmi'
+	icon_state = "tongue"
+	greyscale_config = /datum/greyscale_config/mutant_organ
+	greyscale_colors = GONDOLA_COLORS
+	organ_traits = TRAIT_MUTE
+
+/obj/item/organ/internal/tongue/gondola/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/noticable_organ, "mouth is permanently affixed into a relaxed smile.", BODY_ZONE_PRECISE_MOUTH)
+	AddElement(/datum/element/organ_set_bonus, /datum/status_effect/organ_set_bonus/gondola)
+
+/obj/item/organ/internal/tongue/gondola/Insert(mob/living/carbon/tongue_owner, special, drop_if_replaced)
+	. = ..()
+	tongue_owner.add_mood_event("gondola_zen", /datum/mood_event/gondola_serenity)
+
+/obj/item/organ/internal/tongue/gondola/Remove(mob/living/carbon/tongue_owner, special)
+	tongue_owner.add_mood_event("gondola_zen", /datum/mood_event/gondola_serenity)
+	. = ..()
+
+/// Loving arms: your hands become unable to hold much of anything but your hugs now infuse the subject with pax.
+/obj/item/organ/internal/liver/gondola
+	name = "mutated gondola-liver"
+	desc = "Gondola DNA infused into what was once a normal liver."
+	icon = 'icons/obj/medical/organs/infuser_organs.dmi'
+	icon_state = "liver"
+	greyscale_config = /datum/greyscale_config/mutant_organ
+	greyscale_colors = GONDOLA_COLORS
+	/// instance of the martial art granted on insertion
+	var/datum/martial_art/hugs_of_the_gondola/pax_hugs
+
+/obj/item/organ/internal/liver/gondola/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/organ_set_bonus, /datum/status_effect/organ_set_bonus/gondola)
+	AddElement(/datum/element/noticable_organ, "left arm has small needles breaching the skin all over it.", BODY_ZONE_L_ARM)
+	AddElement(/datum/element/noticable_organ, "right arm has small needles breaching the skin all over it.", BODY_ZONE_R_ARM)
+	pax_hugs = new
+
+/obj/item/organ/internal/liver/gondola/Destroy()
+	QDEL_NULL(pax_hugs)
```

Comment:
Do we have to delete the martial art here?

Reply:
suppose not